Bookish/Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@model Bookish.Web.Models.CatalogueModel
+@{
+    ViewData["Title"] = "Catalogue";
+}
+
+<h1>Catalogue</h1>
+<p>These are all the books in our system</p>
+
+<div>
+    <table class="table table-striped">
+        <thead class="thead-dark">
+        <tr>
+            <th>Title</th>
+            <th>Author(s)</th>
+            <th>ISBN</th>
+            <th></th>
+        </tr>
+        </thead>
+        
+        @foreach (var book in Model.Books)
+        {
+            <tr>
+                <td>@book.title</td>
+                <td>@book.authors</td>
+                <td>@book.isbn</td>
+                <td>
+                    @Html.ActionLink("Borrow","Book", "Home")
```

Comment:
The most common way to add links is using `<a>` tag (or button if you want a button) with `asp-action` set to the action you want to use 🙂